analyzers/tests/SonarAnalyzer.Test/TestCases/BooleanLiteralUnnecessary.Fixed.cs

```diff
@@ -34,15 +34,15 @@ public BooleanLiteralUnnecessary(bool a, bool b, bool? c, Item item)
             var x = false;                  // Fixed
             x = true;              // Fixed
             x = true;                     // Fixed
-            x = a;                    // Fixed
+            x = !a;                    // Fixed
             x = !a;                    // Fixed
 
             x = a;                // Fixed
             x = a;                  // Fixed
             x = a;                  // Fixed
             x = a;                 // Fixed
             x = !a;                  // Fixed
-            x = a;                 // Fixed
+            x = !a;                 // Fixed
```

Comment:
Good luck to the reviewer of this.
It is quite hard to find the correct line this matches to.
Tip: If the `before` file has `// ^^^`, count one _less_ line in the `Fixed` file.